prime-router/metadata/HL7/catchall/hl7/datatypes/PLN/Identifier.yml

```diff
@@ -0,0 +1,49 @@
+# $schema: ./../../../../../json_schema/fhir/hl7-to-fhir-mapping-resource-template.json
+
+resourceType: Identifier
```

Comment:
The PLN[Identifier] mapping needed to be implemented to support the PRT segment.